src/app/components/core/control-dispatcher/add-dispatcher-service/add-dispatcher-service.component.ts

```diff
@@ -16,11 +15,11 @@ export class AddDispatcherServiceComponent implements OnInit {
   dispatcherServiceInstalled;
   dispatcherServiceAdded;
   dispatcherServiceEnabled;
+  dispatcherServiceName: string = '';
   showConfigureModal: boolean = false;
```

Comment:
Just for FYI. There is no need to explicitly telling the type of the variable if you are passing the default value as typescript automatically convert the type based on the default value. Like here no need to give type boolean as you are passing default that is a boolean itself.
`showConfigureModal = false;` // typecast automatically into boolean
dispatcherServiceName = ''; // typecast automatically into string

Reply:
Thanks for the clarification but I already know about it. I believe that there is no harm in that form also but if you suggest to not write in this way then I'll take care of it.